remix.init/setup-easy-auth.js

```diff
@@ -0,0 +1,159 @@
+    message: "Enter client Secret:",
+    type: "input"
+  }]);
+}
+
+async function renderFacebookInstructions(url) {
+  console.log(`Homepage URL: ${url}`)
+  console.log(`Authorization callback URL: ${url}/.auth/login/facebook/callback`)
+  return await inquirer.prompt([{
+    name: "clientId",
+    message: "Enter app ID:",
+    type: "input"
+  }, {
+    name: "clientSecret",
+    message: "Enter app Secret:",
+    type: "input"
+  }]);
+}
+
+async function renderGitHubInstructions(url) {
+  console.log("Follow the instructions for creating an OAuth app on GitHub.")
+  console.log("Instructions: https://docs.github.com/en/developers/apps/building-oauth-apps/creating-an-oauth-app")
+  console.log("You will need to enter below information.")
+  console.log(`Homepage URL: ${url}`)
+  console.log(`Authorization callback URL: ${url}/.auth/login/github/callback`)
+  console.log("On the application page, make note of the Client ID, which you'll need later.")
+  console.log("Under Client Secrets, select Generate a new client secret.")
+  console.log("Make note of the client secret value, which you'll need later.")
+  return await inquirer.prompt([{
+    name: "clientId",
+    message: "Enter client ID:",
+    type: "input"
+  }, {
+    name: "clientSecret",
+    message: "Enter client Secret:",
+    type: "input"
+  }]);
+}
+
+async function renderGoogleInstructions(url) {
+  console.log("Follow the instructions for creating an OAuth app on GitHub.")
+  console.log("Instructions: https://developers.google.com/identity/protocols/oauth2/openid-connect")
+  console.log(`Homepage URL: ${url}`)
+  console.log(`Authorization callback URL: ${url}/.auth/login/google/callback`)
+  return await inquirer.prompt([{
+    name: "clientId",
+    message: "Enter client ID:",
+    type: "input"
+  }, {
+    name: "clientSecret",
+    message: "Enter client Secret:",
+    type: "input"
+  }]);
+}
+
+async function renderTwitterInstructions(url) {
+  console.log(`Homepage URL: ${url}`)
+  console.log(`Authorization callback URL: ${url}/.auth/login/twitter/callback`)
+  return await inquirer.prompt([{
+    name: "clientId",
+    message: "Enter API key:",
+    type: "input"
+  }, {
+    name: "clientSecret",
+    message: "Enter API Secret:",
+    type: "input"
+  }]);
+}
+
+async function prepareSetupProvider(subscriptionId, location, resourceGroup, appName) {
+  await execSync(`az account set --subscription ${subscriptionId}`)
+  await execSync(`az config set defaults.location=${location} defaults.group=${resourceGroup}`)
+  await execSync(`az webapp auth update --name ${appName} --enabled true --action AllowAnonymous`)
+
+  return async function (provider, clientId, clientSecret) {
+    await execSync(`az webapp auth ${provider} update --name ${appName} --client-id ${clientId} --client-secret ${clientSecret} --yes`)
+  }
+}
+
+module.exports = setupEasyAuth
+
+
+// az webapp auth update --resource-group chiptunestac9c98 --name chiptunestac9c98
```

Comment:
Ska vi plocka bort denna kommenterade kod?

Reply:
fixed in [remove comments](https://github.com/FrimJo/chiptune-stack/pull/1/commits/ccb9a3a5ad6d9b6144c6010f56aebc7e16397e0a).